azure-pipelines.yml

```diff
@@ -17,9 +17,16 @@ jobs:
            apt-get -y install python3.6-dev python3-pip locales && \
            locale-gen fr_FR && update-locale && \
            pip3 install setuptools nose cython==0.29.0 pytest pytz pickle5 && \
-           apt-get -y install libopenblas-dev gfortran && \
+           apt-get -y install gfortran-5 wget && \
+           cd .. && \
+           mkdir openblas && cd openblas && \
+           wget https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-v0.3.4-manylinux1_i686.tar.gz && \
```

Comment:
I think yml will let you break these long lines with `\`. The effect seems to be to ignore all the following whitespace, including at the beginning of the next line. Don't need to do that here, but might want to experiment next time.